draft-iab-path-signals-collaboration.md

```diff
@@ -207,6 +207,12 @@ interoperability and so on.
 This section attempts to provide some architecture-level principles
 that would help future designers and recommend useful models to apply.
 
+While RFC 8558 {{RFC8558}} is focusing specifically on "on-path elements",
+the principles described in this document are effectively applicable to
+all kind of information exposure and distribution, not matter if an
+involved node is considered as an "end" or otherwise explicitly addressed,
```

Comment:
```suggestion
involved node is considered as an "end" or any kind of network control element
that is explicitly addressed in the communication,
```
```suggestion
involved node is considered as an "end" or otherwise explicitly addressed,
```